### DISCUSSION TITLE
Curio Display price checker

### DISCUSSION THREAD

**Discussion by Igneaalis** [SnosMe/awakened-poe-trade#785] [Ideas]
In my opinion it would be a huge QoL to have a Heist Curio Display price checker instead of manual searching in the trade website. How do you think?

**Comment by SnosMe** [comment]
It would, but idk how well OCR works

This is what you want
https://www.reddit.com/r/pathofexile/comments/v58xpx/introducing_a_price_checker_for_alternate_quality/

APT only has manual way for price-checking in Overlay

![image](https://user-images.githubusercontent.com/4292308/205918571-d83fd500-c9e6-430d-8a43-8520e9e28f25.png)
